slick.grid.js

```diff
@@ -4112,12 +4112,16 @@ if (typeof Slick === "undefined") {
       for (var row in rowsCache) {
         for (var i in rowsCache[row].rowNode) {
           if (rowsCache[row].rowNode[i] === rowNode)
-          return (row ? parseInt(row) : 0);
+          return parseRowToNumber(row);
         }
       }
       return null;
     }
 
+    function parseRowToNumber(row) {
+      return (row ? parseInt(row) : 0);
```

Comment:
I would prefer to see something like the following line, the reason is that calling `parseInt` on something that is not a number is not safe and could throw an error
```js
return isNaN(row) ? 0 : parseInt(row);
```

Reply:
AFAIK NaN is a 'falsey' value in javascript, so in that case a zero will be returned by the switch.

Reply:
The code I suggested is roughly the same at the original one, I thought that `parseInt` was throwing an error when trying to parse invalid numbers (like 'abc') but it actually returns `NaN` instead of throwing. That is better than I thought. Anyhow, this code is no longer in the PR anyway